### PR TITLE
BUG Re-enable test coverage in travis build

### PR DESCRIPTION
**Pull request by maxime-rainville**
# Parent issue
* https://github.com/silverstripe/silverstripe-assets/issues/389

### CODE REVIEW

**Comment by bergice** [comment]
@maxime-rainville is this still a draft?

**Comment by maxime-rainville** [comment]
Yes ... I haven't figured out the solution yet. I just created the PR so we would have a good way to see the error.

**Comment by emteknetnz** [comment]
This pull request hasn't had any activity for a while. Are you going to be doing further work on it, or would you prefer to close it now?

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ before_script:
   - phpenv config-rm xdebug.ini
   - export PATH=~/.config/composer/vendor/bin:$PATH
   - echo 'memory_limit = 3G' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - php -m
```
Comment (maxime-rainville) [inline .travis.yml:36]:
```suggestion
```

**Comment by maxime-rainville** [comment]
I'm using a PCOV to build the code coverage instead of `phpdbg`. It's meant to be a lot faster and a lot more memory efficient.

There's [some interesting benchmark](https://medium.com/@nicocabot/speed-up-phpunit-code-coverage-analysis-4e35345b3dad) that seems to indicate it's only slightly less efficient than running plain phpunit test.

That's the tutorial I was following to get it set up: https://dev.to/swashata/setup-php-pcov-for-5-times-faster-phpunit-code-coverage-3d9c

**Comment by emteknetnz** [comment]
Nice one, good job

Timely as well, with PHP 8 on the horizon there may be a bunch of changes we want to roll out to travis at the same time, such as https://github.com/silverstripe/silverstripe-framework/issues/9672

**Pull request merged**